doc/src/vpr/command_line_usage.rst

```diff
@@ -182,6 +182,15 @@ General Options
 
      **Default:** ``ideal``
 
+.. option:: --two_stage_clock_routing
+
+    Routes clock nets in two stages using a dedicated clock network.
+
+     * First stage: From the net source (e.g. an I/O pin) to a dedicated clock network source (e.g. center of chip)
+     * Second stage: From the clock network source to net sinks.
+
```

Comment:
Do we need a comment around here stating that the two stage router does not currently work with `--read_rr_graph`?